src/SFA.DAS.ProviderRelationships.UnitTests/Application/Queries/SearchProvidersQueryHandlerTests.cs

```diff
@@ -17,7 +18,7 @@ public class SearchProvidersQueryHandlerTests : FluentTest<SearchProvidersQueryH
         [Test]
         public Task Handle_WhenHandlingASearchProvidersQueryAndAProviderIsFound_ThenShouldReturnASearchProvidersQueryResponse()
         {
-            return RunAsync(f => f.SetProviderResponse(), f => f.Handle(), (f, r) =>
+            return RunAsync(f => f.SetProvider(), f => f.Handle(), (f, r) =>
             {
                 r.Should().NotBeNull();
                 r.Ukprn.Should().Be(int.Parse(f.Query.Ukprn));
```

Comment:
Might as well use `long.Parse` rather than relying on an implicit conversion.
```suggestion
                r.Ukprn.Should().Be(long.Parse(f.Query.Ukprn));
```